src/app/content/components/Content.spec.tsx

```diff
@@ -232,15 +235,15 @@ describe('content', () => {
 
     renderer.act(() => {
       component.root
-        .findByProps({ 'aria-label': 'Click to close the Table of Contents' })
+        .findByProps({ "aria-label": "Click to close the Table of Contents" })
         .props.onClick();
     });
 
     expect(component.root.findByType(TableOfContents).props.isOpen).toBe(false);
 
     renderer.act(() => {
       component.root
-        .findByProps({ 'aria-label': 'Click to open the Table of Contents' })
+        .findByProps({ "aria-label": "Click to open the Table of Contents" })
```

Comment:
A lot of changes from `'` to `"` in this file, not sure if desired seeing that `'` is more commonly used

Reply:
yea i think the snapshot updater did that, i would prefer not to have all the random changes, but i don't care a lot